src/app/tests/suites/commands/delay/DelayCommands.cpp

```diff
@@ -63,3 +63,15 @@ CHIP_ERROR DelayCommands::RunInternal(const char * command)
     VerifyOrReturnError(system(command) == 0, CHIP_ERROR_INTERNAL);
     return ContinueOnChipMainThread(CHIP_NO_ERROR);
 }
+
+CHIP_ERROR DelayCommands::BusyWaitFor(uint32_t durationInMs)
```

Comment:
Could take the ms type as arg and still use the `_ms` suffix at the caller, but either way.

Reply:
I have updated the code to keep the `_ms` suffix.